### DISCUSSION TITLE
VS2022 build失败问题

### DISCUSSION THREAD

**Discussion by voidiii** [BoomingTech/Piccolo#447] [Q&A] [answered]
![92920dd51790320a5df263c64343131](https://user-images.githubusercontent.com/41127515/235838981-3ce763e0-0687-4a0c-9754-870c438efe69.png)
是这行assert失败了：
![fc3ceb4eee838d2fb314a7b50363c47](https://user-images.githubusercontent.com/41127515/235839001-98810baa-ebc3-424b-ac37-4ff33e4b28c5.png)

希望大佬们不吝赐教

**Comment by ShenMian** [comment, marked as answer]
这不属于构建失败, 而是运行时断言失败.  
解决方案: https://github.com/BoomingTech/Piccolo/issues/23#issuecomment-1091037295

**Reply by voidiii** [reply]
感激不尽！

这比论坛效率高多了哈哈哈还是github好
